### PR TITLE
feat(utils): add metrics

### PR DESCRIPTION
**Pull request by lamartire**
@ifiri ready for review

### CODE REVIEW

File: packages/utils/src/metrics.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
+// @ts-check
```
Comment (dudiq) [inline packages/utils/src/metrics.js:1]: maybe make single file for getAddress and getFinger? and not store all in one place
like it done for keystore files

File: packages/utils/src/metrics.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+ * Extracts network address from SDP parameters
+ * @param {string} rawSdp Raw SDP parameters
+ * @returns {string|null}
+ */
+const getAddressFromSDP = rawSdp => {
+  let address = null;
+  const sdpParts = rawSdp.split('\r\n');
+
+  sdpParts.forEach(line => {
+    if (CANDIDATE_ATTR_REG.test(line)) {
```
Comment (dudiq) [inline packages/utils/src/metrics.js:21]: `.forEach` to `.reduce`

Comment (lamartire) [reply packages/utils/src/metrics.js:21]: There are problems with typescript. `forEach` is fitting ok

File: packages/utils/src/metricsTimezone.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+ *   gmt: number|string,
+ * }}
+ */
+const getTimezone = () => {
+  const dateParts = new Date().toString().split(' ');
+  const timezoneType = dateParts[dateParts.length - 2];
+  const timezoneOffset = dateParts[dateParts.length - 1];
+
+  const timezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+  const gmt = `${timezoneType} ${timezoneOffset}`;
```
Comment (dudiq) [inline packages/utils/src/metricsTimezone.js:15]: this code is not working. try to run this part in browser, and you'll see it that `gmt` value is bad.
all you need is:
```
const date = new Date();
return date.getTimezoneOffset();
```

Comment (dudiq) [reply packages/utils/src/metricsTimezone.js:15]: @ifiri or what format should be?

Comment (lamartire) [reply packages/utils/src/metricsTimezone.js:15]: It is not ready yet. I will call you

**Comment by lamartire** [comment]
@dudiq now it is ready for review

File: packages/utils/src/metricsTimezone.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+/**
+ * @returns {{
+ *   timezone: string,
+ *   timezoneOffset: number,
+ * }}
+ */
+const getTimezone = () => {
+  const timezoneOffset = new Date().getTimezoneOffset();
+  const timezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
```
Comment (dudiq) [inline packages/utils/src/metricsTimezone.js:11]: @ifiri is it correct format?
if yes, all approved

**Comment by dudiq** [comment]
@kazurus approved

**Pull request merged**